Collecting ansible-lint<5,>=4.0.2 (from -r D:/Windsor/WindsorThesis/Git repo/Thesis-Project-Python-Dependency-Conflict-Resolution/pipTest/watchman\requirements.125.molecule.txt (line 1))
  Obtaining dependency information for ansible-lint<5,>=4.0.2 from https://files.pythonhosted.org/packages/fb/da/16514c866c4215569399386dae5b8a3ccd531808823c1b0c21fd58bdd982/ansible_lint-4.3.7-py2.py3-none-any.whl.metadata
  Downloading ansible_lint-4.3.7-py2.py3-none-any.whl.metadata (26 kB)
Collecting anyconfig==0.9.7 (from -r D:/Windsor/WindsorThesis/Git repo/Thesis-Project-Python-Dependency-Conflict-Resolution/pipTest/watchman\requirements.125.molecule.txt (line 2))
  Downloading anyconfig-0.9.7.tar.gz (97 kB)
     ---------------------------------------- 97.1/97.1 kB 1.4 MB/s eta 0:00:00
  Installing build dependencies: started
  Installing build dependencies: finished with status 'done'
  Getting requirements to build wheel: started
  Getting requirements to build wheel: finished with status 'error'
  error: subprocess-exited-with-error
  
  Getting requirements to build wheel did not run successfully.
  exit code: 1
  
  [31 lines of output]
  Traceback (most recent call last):
    File "D:\Windsor\WindsorThesis\Git repo\Thesis-Project-Python-Dependency-Conflict-Resolution\pipTest\myenv\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 353, in <module>
      main()
    File "D:\Windsor\WindsorThesis\Git repo\Thesis-Project-Python-Dependency-Conflict-Resolution\pipTest\myenv\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 335, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "D:\Windsor\WindsorThesis\Git repo\Thesis-Project-Python-Dependency-Conflict-Resolution\pipTest\myenv\Lib\site-packages\pip\_vendor\pyproject_hooks\_in_process\_in_process.py", line 118, in get_requires_for_build_wheel
      return hook(config_settings)
             ^^^^^^^^^^^^^^^^^^^^^
    File "C:\Users\ASUS\AppData\Local\Temp\pip-build-env-5klc06wy\overlay\Lib\site-packages\setuptools\build_meta.py", line 327, in get_requires_for_build_wheel
      return self._get_build_requires(config_settings, requirements=[])
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    File "C:\Users\ASUS\AppData\Local\Temp\pip-build-env-5klc06wy\overlay\Lib\site-packages\setuptools\build_meta.py", line 297, in _get_build_requires
      self.run_setup()
    File "C:\Users\ASUS\AppData\Local\Temp\pip-build-env-5klc06wy\overlay\Lib\site-packages\setuptools\build_meta.py", line 497, in run_setup
      super().run_setup(setup_script=setup_script)
    File "C:\Users\ASUS\AppData\Local\Temp\pip-build-env-5klc06wy\overlay\Lib\site-packages\setuptools\build_meta.py", line 313, in run_setup
      exec(code, locals())
    File "<string>", line 12, in <module>
    File "C:\Users\ASUS\AppData\Local\Temp\pip-install-2fnm8tr9\anyconfig_e1fba560ce6041e8b346dd889b4fd58f\anyconfig\__init__.py", line 20, in <module>
      from .api import (
    File "C:\Users\ASUS\AppData\Local\Temp\pip-install-2fnm8tr9\anyconfig_e1fba560ce6041e8b346dd889b4fd58f\anyconfig\api.py", line 77, in <module>
      import anyconfig.backends
    File "C:\Users\ASUS\AppData\Local\Temp\pip-install-2fnm8tr9\anyconfig_e1fba560ce6041e8b346dd889b4fd58f\anyconfig\backends.py", line 16, in <module>
      import anyconfig.ioinfo
    File "C:\Users\ASUS\AppData\Local\Temp\pip-install-2fnm8tr9\anyconfig_e1fba560ce6041e8b346dd889b4fd58f\anyconfig\ioinfo.py", line 16, in <module>
      import anyconfig.utils
    File "C:\Users\ASUS\AppData\Local\Temp\pip-install-2fnm8tr9\anyconfig_e1fba560ce6041e8b346dd889b4fd58f\anyconfig\utils.py", line 385, in <module>
      _LIST_LIKE_TYPES = (collections.Iterable, collections.Sequence)
                          ^^^^^^^^^^^^^^^^^^^^
  AttributeError: module 'collections' has no attribute 'Iterable'
  [end of output]
  
  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

Getting requirements to build wheel did not run successfully.
exit code: 1

See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

[notice] A new release of pip is available: 23.2.1 -> 24.1.2
[notice] To update, run: python.exe -m pip install --upgrade pip
